gk/main.c

```diff
@@ -654,6 +759,13 @@ cleanup_gk(struct gk_config *gk_conf)
 	int i;
 
 	for (i = 0; i < gk_conf->num_lcores; i++) {
+		unsigned socket = rte_lcore_to_socket_id(gk_conf->lcores[i]);
+		if (ipv6_if_configured(&gk_conf->net->front) &&
+				gk_conf->ipv6_acls[socket] != NULL) {
+			rte_acl_free(gk_conf->ipv6_acls[socket]);
+			gk_conf->ipv6_acls[socket] = NULL;
```

Comment:
Having ACLs allocated per socket is a good idea since this enables each socket to quickly access the ACLs. This is still a good idea even accounting for the work to keep these ACLs identical.

You should think of the IPv6 filter in the same way we think of filter in the hardware: if there are conflicts or the filter is wrongly defined, it's a bug.